workspaces/apps-script-research-agent/agent_context.yaml

```
@@ -0,0 +1,43 @@
+  - Error code reference with handling strategies
+  - Rate limiting guidelines documented
+  - Example API calls with responses
+  - Integration pattern recommendations
+
+technical_context:
+  api_docs: "https://developers.google.com/apps-script/api/reference/rest"
+  key_endpoints:
+    - "GET /v1/projects/{scriptId}/content"
+    - "GET /v1/projects/{scriptId}"
+  rate_limits: "100 requests per 100 seconds per user"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)